HerPublicWebsite/appsettings.Production.json

```diff
@@ -1,4 +1,9 @@
 {
+    "BasicAuth": {

```

Comment:
As discussed this is OK, as this username/password is just to stop people visiting the site accidentally, rather than to provide any actual security.